core/modules/node/config/views.view.recent_content.json

```diff
@@ -0,0 +1,559 @@
+{
+    "_config_name": "views.view.recent_content",
+    "name": "recent_content",
+    "description": "Recent content.",
```

Comment:
Lets retain the text from the old block description:
```suggestion
    "description": "A list of recently published content.",
```

![image](https://github.com/backdrop/backdrop/assets/2423362/a8c754c5-f9fe-464c-8717-3ad8552af799)

Reply:
Done